utils/host/src/logger.rs

```diff
@@ -63,18 +64,32 @@ pub fn setup_logger() {
         let resource = Resource::new(params);
         global::set_text_map_propagator(TraceContextPropagator::new());
 
-        let env_filter = build_env_filter();
-        // Set up logging using the provided format
-        let fmt_layer = tracing_subscriber::fmt::layer()
-            .with_level(true)
-            .with_target(false)
-            .with_thread_ids(false)
-            .with_thread_names(false)
-            .with_file(false)
-            .with_line_number(false)
-            .with_filter(env_filter);
-
-        let env_filter = build_env_filter();
+        let log_format = std::env::var("LOG_FORMAT").unwrap_or("pretty".to_string());
+        let fmt_layer: Option<Box<dyn Layer<_> + Send + Sync>> =
+            match log_format.to_lowercase().as_str() {
+                "json" => {
+                    // Initialize with JSON formatting
+                    Some(Box::new(
+                        tracing_subscriber::fmt::layer()
+                            .event_format(tracing_subscriber::fmt::format().json())
+                            .with_filter(build_env_filter()),
+                    ))
+                }
+                "pretty" | _ => {
+                    // Default to pretty formatting with ANSI colors
+                    Some(Box::new(
+                        tracing_subscriber::fmt::layer()
+                            .with_level(true)
+                            .with_target(false)
+                            .with_thread_ids(false)
+                            .with_thread_names(false)
+                            .with_file(false)
+                            .with_line_number(false)
+                            .with_ansi(true)
```

Comment:
Let's avoid defaulting to ANSI colors since it makes very noisy for log collectors like CloudWatch.

Instead, let's add ansi feature and allow users to set based on `NO_COLOR` env var.
```
let ansi = cfg!(feature = "ansi") && env::var("NO_COLOR").map_or(true, |v| v.is_empty());
```

It looks like this in the cloud watch without `NO_COLOR` being set.
```
[2m2025-06-19T06:59:03.321619Z[0m [32m INFO[0m Database configured successfully. |  
```